scripts/upgrade

```diff
@@ -113,5 +114,12 @@ popd
 #=================================================
 # RELOAD NGINX
 #=================================================
-systemctl restart $app
+
 systemctl reload nginx
+
+#=================================================
+# START SERVICE
+#=================================================
+
+ynh_systemd_action --action=start --service_name=$app --log_path="systemd" 
```

Comment:
So eh, sorry if I already asked this somewhere else .. but shouldn't be a restart here ?

Reply:
yes better to user restart than start, as service wekan has not been stoped